File: Beecrowd/1 - Beginner/2003 - 2176/2140 - Two Bills/2140_TwoBills.py
while True:
    buy_price, sell_price = map(int, input().split())

    if buy_price == 0 and sell_price == 0:
        break

    change = sell_price - buy_price

    if change in [7, 12, 22, 52, 102, 15, 25, 55, 105, 30, 60, 110, 70, 120, 150]:
        print("possible")
    else:
        print("impossible")

File: Beecrowd/1 - Beginner/2003 - 2176/2140 - Two Bills/2140_TwoBills.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int buyPrice, sellPrice;

    while (cin >> buyPrice >> sellPrice) {
        if (buyPrice == 0 && sellPrice == 0) break;

        int change = sellPrice - buyPrice;

        if (change == 7 || change == 12 || change == 22 ||
            change == 52 || change == 102 || change == 15 ||
            change == 25 || change == 55 || change == 105 ||
            change == 30 || change == 60 || change == 110 ||
            change == 70 || change == 120 || change == 150) {
            cout << "possible" << endl;
        } else cout << "impossible" << endl;
    }

    return 0;
}